curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-drum-machine.md

```diff
@@ -28,6 +28,26 @@ You can use any mix of HTML, JavaScript, CSS, Bootstrap, SASS, React, Redux, and
 
 **User Story #7:** When a `.drum-pad` is triggered, a string describing the associated audio clip is displayed as the inner text of the `#display` element (each string must be unique).
 
+**Use the following audio clips for your project:**
```

Comment:
This verbiage implies that campers *have* to use this audio, which is not the case.

I'd prefer something like:

```suggestion
Here are some audio samples you can use for your drum machine:
```